gst.so/src/gst_worker.cpp

```diff
@@ -542,9 +543,19 @@ void GSTWorker::run() {
             " Starting the GST stress test "; 
     rvs::lp::Log(msg, rvs::logtrace);
 
+    // log GST ramp up - start message
+    msg = "[" + action_name + "] " + MODULE_NAME + " " +
+            std::to_string(gpu_id) + " Start of GPU ramp up";
+    rvs::lp::Log(msg, rvs::logresults);
+
     // let the GPU ramp-up and check the result
     bool ramp_up_success = do_gst_ramp(&error, &err_description);
```

Comment:
can we use the return value too to determine success/failure?

Reply:
Currently, we have not tuned ramp up time (using default time now) yet. Better not to provide ramp-up as a result which might cause ambiguity even though most of the time it will be SUCCESS.